packages/clay-table/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "clay-table",
+  "version": "1.0.0-alpha.8",
+  "description": "Metal ClayTable component",
+  "license": "BSD",
+  "repository": "https://github.com/metal/metal-clay-components/tree/master/packages/clay-table",
+  "engines": {
+    "node": ">=0.12.0",
+    "npm": ">=3.0.0"
+  },
+  "main": "lib/ClayTable.js",
+  "esnext:main": "src/ClayTable.js",
+  "jsnext:main": "src/ClayTable.js",
+  "files": [
+    "lib",
+    "src",
+    "test"
+  ],
+  "scripts": {
+    "build": "npm run soy && webpack",
+    "compile": "babel -d lib/ src/ -s --ignore src/__tests__",
+    "prepublish": "npm run soy && npm run compile",
+    "soy": "metalsoy --soyDeps '../../node_modules/clay-+(button|checkbox|dropdown|icon|label|link|progress-bar|radio)/src/**/*.soy'"
+  },
+  "keywords": [
+    "clay",
+    "metal"
+  ],
+  "dependencies": {
+    "clay-button": "1.0.0-alpha.8",
```

Comment:
hey @carloslancha, Upgrade to the latest versions of Clay to not have some problems in compiling and jest.

Reply:
Thx @matuzalemsteles I didn't realize versions changed while developing the component ;)